### DISCUSSION TITLE
Triage "needs triage" PRs on the next triage meeting

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#15827] [General]
There are two ["needs triage" PRs](https://github.com/OrchardCMS/OrchardCore/labels/needs%20triage) currently (might be one in a bit), where the original reviewer asks for a second opinion (there were a couple more earlier today, but I removed those that were actually triaged, or are otherwise not ready for a triage). Please, check them out and approve/reject at the next triage meeting.

@agriffard @MikeAlhayek @sebastienros @Skrypt and whoever else will take part.

**Comment by hishamco** [comment]
IMHO we need to focus on and revise the old issues & PRs and leave them of course, which will make the authors upset. Also we can prioritize them so we can easy to triage them every week

**Reply by Piedone** [reply]
All PRs got attention, that's solved now. It's that the PRs with this label also need a second opinion. I'm going through old issues under https://github.com/OrchardCMS/OrchardCore/issues/15034.
